gramex/apps/ide/ide.py

```diff
@@ -0,0 +1,85 @@
+"""
+ Gramex config editing through IDE
+ Entry point is "ide_config_handler"
+"""
+import json
+import yaml
+from functools import reduce
+import operator
+from gramex.config import app_log
+
+node_id = 0         # counter to generate node id's while creating JSON from YAML
+
+def recursive_parse_dict_create_json(d, pid, new_d, config_list):
+    global node_id
```

Comment:
Let's explore how to remove the global

Reply:
Have already done it and also added the scenario we discussed in the morning.